modules/ROOT/partials/misc/supported_versions.adoc

```diff
@@ -6,14 +6,7 @@ Supported versions of {productname}:
 [cols="^,^,^",options="header"]
 |===
 |Version |Release Date |End of Support
-|6.0 |TBD |TBD
-|5.9 |September 15, 2021 |TBD
-|5.8 |May 19, 2021 |November 19, 2022
-|5.7 |February 24, 2021 |August 24, 2022
-|5.6 |December 8, 2020 |June 8, 2022
-|5.5 |October 21, 2020 |April 21, 2022
-|5.4 |July 23, 2020 |January 23, 2022
-|5.3 |June 11, 2020 |December 11, 2021
+|6.0 |April 06, 2022 |TBD
```

Comment:
We probably should leave this as TBD now, especially if we're doing another prod push later this week as we don't want a fixed date advertised yet.